frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -5,6 +5,7 @@ import { useBackend } from "main/utils/useBackend";
 import Accordion from 'react-bootstrap/Accordion';
 import TestJobForm from "main/components/Jobs/TestJobForm";
 import JobComingSoon from "main/components/Jobs/JobComingSoon";
+import JobMilkCows from "main/components/Jobs/JobMilkCows";
```

Comment:
For consistency, we should probably have a consistent naming convention between "TestJobForm" and "MilkCowsJobForm".   Eventually, this component, will have to become a form in order to be functional.   Could you rename it?